content/blog/2025-01-09-AGOF_v2.adoc

```diff
@@ -0,0 +1,363 @@
+frequently requested from Ansible Edge GitOps because they wanted to change it from configuring virtual machines in
+AWS to use actual hardware nodes (for example). It would also be possible to have effectively an empty inventory and
+discover nodes automatically (as earlier iterations of Ansible Edge GitOps do).
+
+The practical consequence of this is that a model pattern using this scheme that runs under OpenShift will
+involve five or more repositories:
+
+1. The OpenShift pattern repository
+2. The AGOF repository which is used to load the AAP configuration
+3. The configuration-as-code repository that defines the objects to be created and maintained in Ansible
+Automation Platform for the pattern
+4. One or more collection repositories which must at minimum contain playbooks to use as Job Templates
+5. One or more inventory repositories to define the nodes on which the pattern will operate.
+
+== Charts for Ansible Validated Patterns
+
+Charts particularly for use in Ansible Patterns:
+
+* https://github.com/validatedpatterns/aap-instance-chart[ansible-automation-platform-instance]
+
+This installs an instance of AAP on the OpenShift cluster, and configures the components. By default it includes
+Controller and Event-Driven Ansible but can also be configured to include Automation Hub.
+
+* https://github.com/validatedpatterns/aap-config-chart[aap-config]
+
+This chart is the one that actually embeds AGOF into the pattern.
+
+* https://github.com/validatedpatterns/openshift-virtualization-chart[openshift-virtualization-instance]
+
+This chart installs and configures an instance of OpenShift Virtualization.
+
+* https://github.com/validatedpatterns/openshift-data-foundations-chart[openshift-data-foundations]
+
+This chart installs and configures an instance of OpenShift Data Foundations suitable for hosting virtual
+machines on.
+
+* https://github.com/validatedpatterns/edge-gitops-vms-chart[edge-gitops-vms]
+
+This chart is responsible for actually creating virtual machines in OpenShift Virtualization. Its defaults
+assume that it is being run on AWS, and that VM data disks will be backed by ODF.
+
+In additiona, Ansible Patterns can use these other charts that are used in several other validated patterns:
+
+* https://github.com/validatedpatterns/hashicorp-vault-chart[hashicorp-vault]
+
+Installs and configures the community edition of Hashicorp Vault. Vault is the default secret storage mechanism
+for Validated Patterns.
+
+* https://github.com/validatedpatterns/golang-external-secrets-chart[golang-external-secrets]
+
+Installs and configures Golang External Secrets. External Secrets is the normal mechanism Validated Patterns
+uses to retrieve and use secrets within the patterns.
+
+== In Action: Federated Edge Observability
+
+Please see the companion blog article that provides a detailed walkthrough of the Federated Edge Observability
```

Comment:
Should we link this companion article or will it appear at a later that? Maybe specify that it will show up on the blog in that case?

Reply:
I thought this whole (writing) project was going to a bit faster. I am still planning on writing it, but haven't started it. My intention at that point was to include the link in this article then.